plugins/LilyPad/LilyPad.cpp

```diff
@@ -163,7 +163,7 @@ struct Stick {
 
 // Sum of states of all controls for a pad (Not including toggles).
 struct ButtonSum {
-	int buttons[12];
+	int buttons[16];
 	Stick sticks[3];
```

Comment:
The d-pad on the oringal DS2 also acts as a stick that makes it impossible to push left and right/up and down at the same time, so I didn't want to mess with it.

I didn't realize it would impact save state. I could just leave the original 12 buttons and work with buttons 5-8 as those aren't used by the dance pad.